frontend/templates/frontend/safe_mode_ui.html

```diff
@@ -1,16 +1,15 @@
+{% load i18n %}
 {% if is_safe_mode_enabled %}
 <div class="safe-mode-disable">
   <div class="container">
     <div class="notification is-primary">
       <form method="post" action="{% url 'safe_mode:disable' %}">
         <div class="columns">
           <div class="column is-8">
-            This site is currently in <strong>compatibility mode</strong>. For an enhanced
-            experience, you can disable it, but this may cause compatibility issues
-            with your current browser.
+            {% blocktrans %}This site is currently in <strong>compatibility mode</strong>. For an enhanced experience, you can disable it, but this may cause compatibility issues with your current browser.{% endblocktrans %}
```

Comment:
[This is how Django does i18n in its templates.](https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#blocktrans-template-tag)  I removed the extra newlines/whitespace from the text so localizers wouldn't be inconvenienced or get confused.